### PR TITLE
[STY] Switched to F-strings

### PR DESCRIPTION
**Pull request by akimbler**
* Switched to `f-strings` from `format()` statements

### CODE REVIEW

File: bidsify/bidsify.py [hunk 1]
```diff
@@ -89,16 +88,16 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
             wk_file.write('.heudiconv/\ntmp/\nvalidator.txt\n')
 
     # Run heudiconv
-    cmd = f'heudiconv {dir_type} {dicom_dir} \
-            -s {subject} -f {heuristics} -c dcm2niix \
-            -o {output_dir} --bids --overwrite --minmeta'
+    cmd = (f'heudiconv {dir_type} {dicom_dir}'
```
Comment (tsalo) [inline bidsify/bidsify.py]: f the whole thing (outside the parens)

**Comment by tsalo** [comment]
I don't think the entry point was missing. It looks like it was just moved down several lines in the file. Was it not working?

File: bidsify/bidsify.py (hunk 1 above)
Comment (akimbler) [reply bidsify/bidsify.py]: That doesn't work in any IDE I've tried it in.

**Comment by akimbler** [comment]
> I don't think the entry point was missing. It looks like it was just moved down several lines in the file. Was it not working?

That might have just been on my end, I'll drop it from the title

File: bidsify/bidsify.py (hunk 1 above)
Comment (tsalo) [reply bidsify/bidsify.py]: well darn. nvm then it's all good.

File: bidsify/bidsify.py [hunk 2]
```diff
@@ -89,16 +89,16 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
...
+           f'-s {subject} -f {heuristics} -c dcm2niix'
+           f'-o {output_dir} --bids --overwrite --minmeta')
     run(cmd, env={'TMPDIR': tmp_path.name})
 
     # Run defacer
     anat_files = sub_dir.glob('/anat/*.nii.gz')
     for anat in anat_files:
-        cmd = f'mri_deface {anat} /src/deface/talairach_mixed_with_skull.gca \
-               /src/deface/face.gca {anat}'
+        cmd = (f'mri_deface {anat} /src/deface/talairach_mixed_with_skull.gca'
```
Comment (tsalo) [inline bidsify/bidsify.py]: needs a trailing space I think

File: bidsify/bidsify.py [hunk 3]
```diff
@@ -89,16 +89,16 @@ def bidsify_workflow(dicom_dir, heuristics, subject, session=None, output_dir='.
             wk_file.write('.heudiconv/\ntmp/\nvalidator.txt\n')
 
     # Run heudiconv
-    cmd = f'heudiconv {dir_type} {dicom_dir} \
-            -s {subject} -f {heuristics} -c dcm2niix \
-            -o {output_dir} --bids --overwrite --minmeta'
+    cmd = (f'heudiconv {dir_type} {dicom_dir}'
+           f'-s {subject} -f {heuristics} -c dcm2niix'
+           f'-o {output_dir} --bids --overwrite --minmeta')
```
Comment (tsalo) [inline bidsify/bidsify.py]: these need trailing spaces I think. not the last one, just the first two.

**Review by tsalo — CHANGES_REQUESTED** [review]
Just small issues with trailing spaces in the f-strings.

**Review by tsalo — APPROVED** [review]
LGTM.

**Pull request merged**